packages/testlab/test/unit/misc.test.ts

```diff
@@ -0,0 +1,75 @@
+// Copyright IBM Corp. 2018. All Rights Reserved.
+// Node module: @loopback/testlab
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {expect} from '../../src/expect';
+import {toJSON} from '../../src/misc';
+
+describe('toJSON', () => {
+  it('removes properties set to undefined', () => {
+    const value = toJSON({
+      title: 'a-todo-title',
+      isComplete: undefined,
+    });
+    expectObjectValue(value, {
+      title: 'a-todo-title',
+    });
+  });
+
+  it('handles null', () => {
+    const value = toJSON(null);
+    expectNull(value);
+  });
+
+  it('handles undefined', () => {
+    const value = toJSON(undefined);
+    expectUndefined(value);
+  });
+
+  it('handles numbers', () => {
+    const value = toJSON(123);
+    expectNumericValue(value, 123);
+  });
+
+  it('handles strings', () => {
+    const value = toJSON('text');
+    expectStringValue(value, 'text');
+  });
+
+  it('handles booleans', () => {
+    const value = toJSON(true);
+    expectBooleanValue(value, true);
+  });
+
+  it('handles dates', () => {
+    const value = toJSON(new Date('2018-01-01T00:00:00.000Z'));
+    expectStringValue(value, '2018-01-01T00:00:00.000Z');
+  });
+});
+
+// Helpers to enforce compile-time type checks
```

Comment:
Should I move these helpers to `src/expect.js` and export them as part of testlab's public API?

Reply:
Let's keep the list of helpers internal for now. Too many public helpers will be hard to maintain.